dashboard/app/views/scripts/show.html.haml

```diff
@@ -32,13 +32,15 @@
 
   .row#stats
     .span12#rightside
-      - if @script.trophies
-        = render partial: 'shared/user_stats', locals: { user: current_user, width: '100%', async: true }
-      - else
-        .user-stats-block
-          :javascript
-            dashboard.progress.renderCourseProgress(#{@script.summarize.to_json});
-          = render partial: 'shared/user_stats_key'
+      .user-stats-block
+        :javascript
+          dashboard.progress.renderCourseProgress(#{@script.summarize.to_json});
```

Comment:
btw, i noticed over in https://github.com/code-dot-org/code-dot-org/pull/8555/files#diff-dcf3309483985fc5e509aabf7524c1e4L196 that this function is hitting /api/user_progress/ for signed-out users.